chapters/ch07.asciidoc

```diff
@@ -1419,9 +1419,9 @@ const things = {
   length: 2
 }
 Array.from(things)
-// <- [<div>, <div>, <div>, ...]
+// <- [Object, Object]
```

Comment:
`Array.from(groceries)` still gives `[Object, Object]`.
How about  going back to the current `things` and I change line 1421 to `const [apple, onion] = Array.from(things)`, followed by `console.log(apple.name)` and `// <- "Apple"`?